src/planscape/forsys/generate_projects_for_a_single_scenario.R

```diff
@@ -95,6 +96,7 @@ generate_projects_for_a_single_scenario <- function(
       patchmax_SDW = 1,
       patchmax_EPW = 0.5,
       patchmax_sample_frac = 0.01,
+      stand_threshold = paste(eligibility_field, ">0", sep=""),
```

Comment:
nit: Infix operators should be surrounded by spaces.
https://style.tidyverse.org/syntax.html#infix-operators

Could also use `paste0()`

Reply:
done.